src/parameters/OGGMparameters.jl

```diff
@@ -8,6 +8,7 @@ struct OGGMparameters <: AbstractParameters
     multiprocessing::Bool
     workers::Int64
     ice_thickness_source::String
+    DEM_source::String
```

Comment:
Nice, what are the different options for DEM sources? This should be explained somehwere. I'd also set a warning in case the user sets a source name that doesn't exist. Something like:
```julia
@assert DEM_source == "whatever" || DEM_source == "whatever" ... "Wrong name for DEM source. Options are: "whatever", "whatever"..."
```

Reply:
The availability of different source options are glacier specific and also for some sources you need to registrate an account (e.g. NASADEM). If there is no data available from that specific source OGGM throws a warning and picks the default source for the glacier that is being processed ( default source varies per glacier as well). 

I think throwing a warning as you suggest could be confusing in the case that the user wants to use sources that require registration and are just not available for that specific glacier. Maybe we should point them to the documentation instead? 

See this [OGGM documentation](https://oggm.org/tutorials/stable/notebooks/advanced/dem_sources.html#oggm-provided-datasets) for all the available sources 

How do you think about this?

Reply:
OK, let's leave it like this for now.